### DISCUSSION TITLE
Outdated Version when using latest version

### DISCUSSION THREAD

**Discussion by Dal1ster** [KadeArchive/Kade-Engine#388] [Q&A] [answered]
I just started using the kade engine, but after I press enter, it tells me that the program is outdated, even though I started fresh with the latest version. what seems to be the problem?

**Comment by puyoxyz** [comment, marked as answer]
gamebanana builds are *very* outdated. download from [releases](https://github.com/KadeDev/Kade-Engine/releases) if you are on windows or from appveyor if on a different platform

appveyor -> [![AppVeyor](https://img.shields.io/appveyor/build/KadeDev/Kade-Engine-Windows?label=windows%20build)](https://ci.appveyor.com/project/KadeDev/kade-engine-windows/build/artifacts) [![AppVeyor](https://img.shields.io/appveyor/build/KadeDev/Kade-Engine-Macos?label=macOS%20build)](https://ci.appveyor.com/project/KadeDev/kade-engine-macos/build/artifacts)  [![AppVeyor](https://img.shields.io/appveyor/build/KadeDev/Kade-Engine-Linux?label=linux%20build)](https://ci.appveyor.com/project/KadeDev/kade-engine-linux/build/artifacts) [![AppVeyor](https://img.shields.io/appveyor/build/daniel11420/KadeEngineWeb?label=html5&20build)](https://ci.appveyor.com/project/daniel11420/KadeEngineWeb)

@KadeDev -> maybe remove downloads from the gamebanana and just leave links to github latest release & appveyor latest?
